plugins/content/themepreset/index.js

```diff
@@ -0,0 +1,99 @@
+// LICENCE https://github.com/adaptlearning/adapt_authoring/blob/master/LICENSE
+var ContentPlugin = require('../../../lib/contentmanager').ContentPlugin;
+var Configuration = require('../../../lib/configuration');
+var Database = require('../../../lib/database');
+var Origin = require('../../../lib/application');
+var Usermanager = require('../../../lib/usermanager');
+var Util = require('util');
+
+function ThemePresetContent () {
+}
+Util.inherits(ThemePresetContent, ContentPlugin);
+
+ThemePresetContent.prototype.hasPermission = function (action, userId, tenantId, contentItem, next) {
+  return next(null, true);
+};
+
+ThemePresetContent.prototype.getModelName = function() {
+  return 'themepreset';
+};
+
+ThemePresetContent.prototype.updatePreset = function(presetId, courseId, res, next) {
+  // save to config
+  app.contentmanager.retrieve('config', { _courseId: courseId }, function(err, config) {
+    if (err) return next(err);
+      var delta = config[0];
```

Comment:
Please sort the indentation in this file.

Reply:
Sorted now